gulpfile.babel.js

```diff
@@ -154,6 +154,18 @@ gulp.task('html', () => {
       removeStyleLinkTypeAttributes: true,
       removeOptionalTags: true
     })))
+    .pipe($.if('*.html', $.minifyInline({
+            js: {
+                output: {
+                    comments: true
```

Comment:
IIUC, this *keeps* the comments in the output. I don‘t think a production build should have comments.